ruby-server-example/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Ruby Alpine image (after using bsf dockerfile digest)
+FROM index.docker.io/library/ruby:3.1-alpine@sha256:f9c75043ce289b61cebdd0a16851d5ae68d82bfb912eb7d6ca4f57cc6780ecef AS build
```

Comment:
We need to show the use of our own base image over here, not the docker one. So in the guide when we show how base image is created, we can show the use of that over here.